app/views/admin/comments/_comment.html.erb

```diff
@@ -35,4 +56,47 @@
       <% end %>
     </footer>
   </article>
+
+  <% if defined?(is_individual_comment) && is_individual_comment %>
+    <% privileged_comment_reactions = comment.reactions.privileged_category.select { |reaction| reaction.reactable_type == "Comment" } %>
```

Comment:
See note above about this check maybe not being necessary

Reply:
Removed the select query and updated to 

`<% privileged_comment_reactions = comment.reactions.privileged_category %>`